ci/publish-github-release.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# publish-github-release.sh creates a "release" in the github repo and associates with the build tag
+
+# usage publish-github-release.sh REPO VERSION
+set -o errexit
+set -o pipefail
+
+if [[ -z "${PULUMI_BOT_GITHUB_API_TOKEN:-}" ]]; then
```

Comment:
If this is being run by GitHub Actions we should be able to use the default `GITHUB_TOKEN` rather separate credentials? If it's being run by Travis or similar, this doesn't apply though.

Reply:
This is *currently* still travis and will run as part of the current release script

Reply:
:+1: